Force use of Python 2 even when Python 3 is the default Python interpreter.

Signed-off-by: Carlos Santos <casantos@datacom.ind.br>

--- ./gtest/scripts/fuse_gtest_files.py.orig	2013-09-18 14:48:30.000000000 -0300
+++ ./gtest/scripts/fuse_gtest_files.py	2015-07-22 15:42:53.291591205 -0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # Copyright 2009, Google Inc.
 # All rights reserved.
--- ./scripts/generator/gmock_gen.py.orig	2013-09-18 14:50:15.000000000 -0300
+++ ./scripts/generator/gmock_gen.py	2015-07-22 17:06:51.071815634 -0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # Copyright 2008 Google Inc. All Rights Reserved.
 #
